### PR TITLE
PlanExplore Page

### PR DESCRIPTION
**Pull request by makoscafee**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -100,3 +105,6 @@ oppiaGithubPages.controller('theExplorationGalleryController', function($scope)
 oppiaGithubPages.controller('learnerViewController', function($scope) {
 
 });
+oppiaGithubPages.controller('planningYourExplorationController', function($scope) {
```
Comment (seanlip) [inline app.js]: Out of interest, what are we planning to do with these controllers?

File: pages/planningYourExploration.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!-- MAIN CONTENT -->
+<section id="main_content" class="inner">
+  <h2>
+  	<a id="Planning_Your_Exploration" class="anchor" href="#Planning_Your_Exploration" aria-hidden="true"><span class="octicon octicon-link"></span></a>Planning Your Exploration</h2>
```
Comment (seanlip) [inline pages/planningYourExploration.html]: This has a tab instead of a space

File: pages/planningYourExploration.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  <p>The problem with writing one card that links to five cards, each of which links to five more cards, etc. is that it results in quite a bit of complexity. We recommend that you do not start by doing this. Instead, start by creating something that is, for the most part, linear. You can always add branching and loops later on. This will help you get to a 'complete' exploration as soon as possible. </p>
+  <p>Another suggestion is to spend some time planning your exploration on paper before converting it into a playable Oppia exploration, since it's easier to edit a rough pencil-and-paper draft in a way that still makes it easy to see the big picture. </p>
+  <p>We'd also encourage you to have a look at the <a href="">Design Tips(TODO:Add link)</a> page, which contains a large list of common patterns that you can make use of when writing an exploration. </p>
+  <h2>
+  	<a id="Crafting_Individual_Responses" class="anchor" href="#Crafting_Individual_Responses"  aria-hidden="true"><span class="octicon octicon-link"></span></a>Crafting Individual Responses</a></h2>
```
Comment (seanlip) [inline pages/planningYourExploration.html]: This, too -- and below as well.

File: app.js (hunk 1 above)
Comment (makoscafee) [inline app.js]: If in the future we might need interaction in pages they can help. Otherwise i can remove them if not needed at all.

Comment (seanlip) [inline app.js]: OK, I'd say let's get rid of them, since they currently add complexity and are not needed. As you say, we can add them later.

Comment (makoscafee) [inline app.js]: Done

File: pages/planningYourExploration.html (hunk 2 above)
Comment (makoscafee) [inline pages/planningYourExploration.html]: Done

File: pages/planningYourExploration.html (hunk 3 above)
Comment (makoscafee) [inline pages/planningYourExploration.html]: done

**Pull request merged**